src/database/services/HookService.ts

```diff
@@ -0,0 +1,62 @@
+import { Hook } from '../models'
+
+interface HookConfig {
+    position: number[],
+    current_panel_id: number,
+    next_panel_set_id: number
+}
+
+/**
+ * Perform queries on the 'hooks' table
+ */
+class HookService {
+    /**
+     * Create a new hook
+     * @param {HookConfig} newHook
+     * @returns {object} with position, current_panel_id, and next_panel_id properties
+     */
+    static async createHook(newHook: HookConfig){
+        const {position, current_panel_id, next_panel_set_id} = await Hook?.create({
```

Comment:
`Hook?.create()` - you don't need the null check for this or the following lines. Otherwise, looks fine